### PR TITLE
[13.0][MIG] purchase_landed_cost: Migration to 13.0

### PR DESCRIPTION
**Pull request by ernestotejeda**
Cc @Tecnativa TT27127

### CODE REVIEW

File: purchase_landed_cost/README.rst [hunk 1]
```diff
@@ -40,8 +40,6 @@ distribute them with a lot of methods.
 * Types marked as default are automatically added to each new purchase
   distribution.
 * Management orders shopping expenses associated with one or more entry slips.
-* Upgrade cost price of products based on the costs.
```
Comment (pedrobaeza) [inline purchase_landed_cost/README.rst]: Pass this to the roadmap

**Review by pedrobaeza — APPROVED** [review]
Good enough as data model with computations, and not having the price update is not critical for our project.

Please squash together all the French translations from the same person.

**Review by marcelsavegnago — APPROVED** [review]
LGTM

**Pull request merged**